recipe-client-addon/bin/tc/build_test.sh

```diff
@@ -31,8 +31,11 @@ echo 'ac_add_options --enable-artifact-builds' > ./mozconfig
 mkdir /root/.mozbuild  # Otherwise mach will hang forever asking for permission
 ./mach build
 
-echo 'Running tests'
-xvfb-run ./mach test browser/extensions/shield-recipe-client/
+echo 'Running mochitests'
+xvfb-run ./mach mochitest browser/extensions/shield-recipe-client/browser
```

Comment:
I don't  think this is the right path. I think you want `.../shield-recipe-client/test/browser`, and similar for `unit` below.